Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -202,6 +207,8 @@ try {
                 Write-Host "BuildOrderDepth=$($buildOrder.Count)"
             }
         }
+        Write-Host "Projects to build: $($buildProjects -join ', ')"
```

Comment:
This is output already in `Get-ProjectsToBuild` so I suggest to remove one of the logging (probably the one in `Get-ProjectsToBuild`).